webpack.config.js

```diff
@@ -13,12 +13,15 @@ module.exports = {
       {
         test: /\.jsx?/,
         include: SRC_DIR,
-        loader: 'babel-loader',
-        query: {
-          presets: [ '@babel/preset-react', '@babel/preset-env' ]
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: [ '@babel/preset-react', '@babel/preset-env'],
+            plugins: ['@babel/plugin-proposal-nullish-coalescing-operator', '@babel/plugin-proposal-optional-chaining']
```

Comment:
Question: I'm curious what you are using Babel plugins are for? This is not feedback, just wondering what the use case is.

Does these, in effect, replace ternary operators?

Reply:
I'm hoping for babel to transpile these syntax into web compatible syntax.    Haven't used it yet, but very soon when I start working on input validation.  I use null coalescing and optional chaining heavily as you can condense your lines of code to a single line:

`
value = x?.value
`
is equivalent to 

`
if ( x != null) {
    value = x.value
}
`

Reply:
Very cool, something I will try to check out myself!